networkx/algorithms/shortest_paths/dense.py

```diff
@@ -13,13 +13,22 @@
 def floyd_warshall_numpy(G, nodelist=None, weight="weight"):
     """Find all-pairs shortest path lengths using Floyd's algorithm.
 
+    This algorithm for finding shortest paths takes advantage of
+    matrix representations of a graph and works well for dense
+    graphs where all-pairs shortest path is desired. The results
```

Comment:
Maybe a tiny typo in here?  "an all-pairs shortest path" or "all-pairs shortest paths"?